tests/python/gpu/test_kvstore_gpu.py

```diff
@@ -91,6 +91,21 @@ def check_rsp_pull(kv, count, ctxs, is_same_rowid=False, use_slice=False):
     check_rsp_push_pull('device')
     check_rsp_push_pull('device', is_push_cpu=False)
 
+
+def test_row_sparse_pull_single_device():
```

Comment:
The test is in kvstore gpu section but uses local kvstore and cpu context. It would probably make sense to change the test here to `device` and `gpu`?